lib/fluent/plugin/out_http.rb

```diff
@@ -120,6 +122,11 @@ def set_text_body(req, data)
     req['Content-Type'] = 'text/plain'
   end
 
+  def set_raw_body(req, data)
+    req.body = data.to_s
+    req['Content-Type'] = 'application/octet-stream'
```

Comment:
`application/octet-stream`  is needed for testing and notify undefined MIME type when using `raw` serializer.
How do you think of this `Content-Type`?

Reply:
> I think this is OK as a default since you can override it with `custom_headers`. For me, I'll set `application/msgpack`.

Ah! It's super cool. :wink: